slack_adapter/slack_helper.py

```diff
@@ -21,4 +24,35 @@ def activity_to_slack(activity: Activity) -> NewSlackMessage:
                 if att.name == 'blocks':
                     message.blocks = [att.content]
                 else:
-                    new_attachment = Object(author_name= att.name, thumb_url=att.thumbnail_url)
\ No newline at end of file
+                    new_attachment = Object(author_name= att.name, thumb_url=att.thumbnail_url)
+
+    @staticmethod
+    def command_to_activity_async(slack_body: SlackRequestBody, client: SlackClientWrapper, cancellation_token):
+        """ Creates an activity based on a slack event related to a slash command
+        :param slack_body: The data of the slack event.
+        :param client: The Slack client.
+        :param cancellation_token: A cancellation token for the task.
+        :return: An activity containing the event data.
+        """
+        if not slack_body:
+            raise Exception("slack_body")

```

Comment:
I'm not sure if this is the proper way to manage exceptions when None is encountered as an argument. I found that most of the python function raises TypeError if None is passed as an argument. Can you check it?